addons/website/views/snippets/s_framed_intro.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_framed_intro" name="Framed Intro">
+    <section class="s_framed_intro pt56 pb56">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item g-col-lg-6 g-height-10 col-lg-6" style="grid-area: 1 / 1 / 11 / 7; z-index: 1; order:2;">
+                    <h1 class="display-4">Experience<br/>the World's Best<br/><strong>Quality Services</strong></h1>
+                </div>
+                <div class="o_grid_item g-col-lg-6 g-height-4 col-lg-6 align-content-end" style="grid-area: 11 / 1 / 15 / 7; z-index: 2; order:3;">
+                    <p class="lead">This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.</p>
+                    <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary">Discover</a>
```

Comment:
I suggest wrapping the `<a>` inside a `<p style="text-algin:left;">` tag so that users can easily control the text alignment.